ajax_select/__init__.py

```diff
@@ -13,6 +13,9 @@
 from django.utils.encoding import force_text
 from django.utils.html import escape
 from django.utils.translation import ugettext_lazy as _
+from django.utils.module_loading import autodiscover_modules
```

Comment:
The user doesn't call autodiscover, it is called by the configuration module, apps.py, by subclassing AppConfig and overriding the ready() method.  The user indirectly calls autodiscover() by using the default configuration.   To use the other configuration without autodiscover(), the user would enter 'ajax_select.SimpleAjaxSelectConfig' in the INSTALLED_APPS dict instead of 'ajax_select'.  The default app configuration is set in the __init__.py.  This is a Django 1.7+ feature (https://docs.djangoproject.com/en/1.8/ref/applications/), so this also breaks 1.5/1.6 compatibility.  I can bring a simpler version of the autodiscover_modules() functionality into the ajax_select app, though, in order to support 1.5/1.6.  I guess I also forgot to mention that this, too, is a 1.5/1.6 breaking feature!